test/python/primitives/test_primitive.py

```diff
@@ -142,7 +142,10 @@ def test_with_scheduling(n):
                 qc = QuantumCircuit(1)
                 qc.x(0)
                 qc.add_calibration("x", qubits=(0,), schedule=custom_gate)
-                return transpile(qc, Fake20QV1(), scheduling_method="alap")
+                backend = GenericBackendV2(
+                    num_qubits=20, basis_gates=["id", "u1", "u2", "u3", "cx"], seed=42
+                )
```

Comment:
This one probably can work with a 2-q backend (avoiding having only 1 qubit not to run into edge cases with the 2q basis gates).
```suggestion
                backend = GenericBackendV2(
                    num_qubits=2, basis_gates=["id", "u1", "u2", "u3", "cx"], seed=42
                )
```

Reply:
2 qubits work